presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/SemiTransactionalHiveMetastore.java

```diff
@@ -961,6 +1049,11 @@ public synchronized void declareIntentionToWrite(ConnectorSession session, Write
         declaredIntentionsToWrite.add(new DeclaredIntentionToWrite(writeMode, hdfsContext, identity, session.getQueryId(), stagingPathRoot, schemaTableName));
     }
 
+    public boolean notYetCommitted()
```

Comment:
Having a negative method name is a bit strange. Maybe use `isFinished` with opposite behavior or call it `isRunning`.

Reply:
Changed to be isFinished with the opposite behavior.